src/video_core/renderer_vulkan/vk_rasterizer.cpp

```diff
@@ -439,7 +456,13 @@ void RasterizerVulkan::Draw(bool is_indexed, bool is_instanced) {
     key.renderpass_params = GetRenderPassParams(texceptions);
     key.padding = 0;
 
-    auto& pipeline = pipeline_cache.GetGraphicsPipeline(key);
+    auto& pipeline = pipeline_cache.GetGraphicsPipeline(key, async_shaders);
+    if (&pipeline == nullptr || pipeline.GetHandle() == VK_NULL_HANDLE) {
```

Comment:
```suggestion
    if (pipeline.GetHandle() == VK_NULL_HANDLE) {
```

References by definition can never be null without invoking undefined behavior.